content/tracing/setup/dotnet.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Tracing .NET Applications
+kind: Documentation
+further_reading:
+- link: "https://github.com/DataDog/dd-trace-csharp"
+  tag: "Github"
+  text: Source code
+- link: "tracing/visualization/"
+  tag: "Documentation"
+  text: "Explore your services, resources and traces"
+---
+
+The APM tracer for .NET applications is currently in **alpha** and not officially supported by Datadog. The officially supported Beta will launch as early as August 2018.  To be notified of the beta launch, [submit this form][1].
```

Comment:
Can we put the form in a button or similar element?  Same on the PHP page. @l0k0ms